docs/superpowers/specs/2026-05-30-character-reference-redesign.md

```
@@ -0,0 +1,177 @@
+  <div className="size-10 rounded-full bg-muted/50 flex items-center justify-center mb-2">
+    <span className="text-xl opacity-40">👥</span>
+  </div>
+  <p className="text-xs text-muted-foreground">No characters defined</p>
+</div>
+```
+
+### Data Flow
+
+**Current flow:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)